code/controllers/subsystem/title.dm

```diff
@@ -4,8 +4,10 @@ SUBSYSTEM_DEF(title)
 	init_order = INIT_ORDER_TITLE
 
 	var/file_path
+	var/lobby_screen_size = "15x15"
```

Comment:
Shouldn't this default to the config entry for default view over a statically assigned 15x15?

Reply:
The area that has a backdrop in the title screen zone is a 15x15 block, so by setting it to 15x15 means that you won't see space and cordon before the lobby screen is loaded. Once the lobby screen loads in, then the lobby_screen_size is calculated by reading the icon sprite.
The title screen initialises 3rd, the only thing that initialises before it is fail2topic and profiler so it should be done before anything else and usually before anyone joins. I could change it to world.view, but then it might make it so you can see too far before the lobby screen loads (which ultimately isnt too bad). I can't change it to the lobby screen size earlier because it gets calculated in initialise.